package.json

```diff
@@ -77,6 +77,7 @@
     "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-qunit": "^4.4.0",
     "ember-cli-testdouble-qunit": "^2.1.1",
+    "ember-compatibility-helpers": "^1.2.0",
```

Comment:
Is this needed?  I don't see a change to the lockfile

Reply:
`ember-compatibility-helpers` is already in the lockfile because a bunch of other packages already depend on it. I think this is why `yarn.lock` has no diff on this branch.

We're now importing from it directly for the new test, hence why it's been added to `package.json`.